src/docfx.website.themes/default/conceptual.html.primary.js

```diff
@@ -3,6 +3,6 @@ var common = require('./common.js');
 
 exports.transform = function (model) {
   model._disableToc = model._disableToc || !model._tocPath || (model._navPath === model._tocPath);
-  model.docurl = model.docurl || common.getViewSourceHref(model);
+  model.docurl = model.docurl || common.getViewSourceHref(model, model.gitContribute);
```

Comment:
It needs the second parameter, see https://github.com/dotnet/docfx/pull/505/files#diff-5bd22e55ddbb69d733cdb3b7284456b2R37. Thus I can link to a different branch when clicking "Improve this Doc" for conceptual doc.